communication-server/package.json

```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "nodemon src/app.js"
+    "start": "node src/app.js",
```

Comment:
Changed this to use node because we don't need a nodemon server to watch for changes. If anyone wants to do development on the communication server you can use `npm run serve`